cmd/world/main.go

```diff
@@ -29,6 +30,11 @@ func init() {
 }
 
 func main() {
+	err := globalconfig.SetupConfigDir()
+	if err != nil {
+		log.Err(err).Msg("could not setup config folder")
+	}
+
```

Comment:
Move this code after sentry initialization (after line 44). so that the sentry error can be sent to the sentry.